docker-compose.yaml

```diff
@@ -1,5 +1,3 @@
-version: "3"
-
 services:
   mkdocs_qne:
```

Comment:
Btw, can we change this name? This seems to refer to the Quantum Network Explorer, right?

Reply:
This is indeed based on that project